pages/configuration/elasticsearch.rst

```diff
@@ -16,6 +16,8 @@ Starting with version 2.3, Graylog uses the HTTP protocol to connect to your Ela
 
 .. caution:: Graylog 2.3 **does not** work with Elasticsearch 6.x yet!
 
+.. note:: Graylog 2.3 works with the `AWS Elasticsearch service <https://aws.amazon.com/elasticsearch-service/>`_ but make sure to use version **5.3** or later.
```

Comment:
Minor nitpick: The official full name is "Amazon Elasticsearch Service", not "AWS Elasticsearch service".